dev-tex/rubber/metadata.xml

```diff
@@ -6,6 +6,6 @@
 		<name>Gentoo TeX Project</name>
 	</maintainer>
 	<upstream>
-		<remote-id type="launchpad">rubber</remote-id>
```

Comment:
Are you [adding](https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Taking_over_an_existing_package) yourself?

Reply:
Right, done in https://github.com/gentoo/gentoo/pull/17722/commits/eedd9a95b066c1ab3da322feab6b72ed9521b40b